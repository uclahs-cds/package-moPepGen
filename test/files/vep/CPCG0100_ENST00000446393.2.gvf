##fileformat=VCFv4.2
##mopepgen_version=0.0.1
##parser=parseVEP
##reference_index=/data/Index/GRCh38-EBI-GENCODE34
##genome_fasta=
##annotation_gtf=
##source=gsnp
##CHROM=<Description='Gene ID'>
##INFO=<ID=TRANSCRIPT_ID,Number=1,Type=String,Description="Transcript ID">
##INFO=<ID=GENE_SYMBOL,Number=1,Type=String,Description="Gene Symbol">
##INFO=<ID=GENOMIC_POSITION,Number=1,Type=String,Description="Genomic Position">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
ENSG00000228336.2	5278	SNV-5278-T-C	T	C	.	.	TRANSCRIPT_ID=ENST00000446393.2;GENOMIC_POSITION=chr1:247775446;GENE_SYMBOL=OR9H1P
ENSG00000228336.2	5435	SNV-5435-C-T	C	T	.	.	TRANSCRIPT_ID=ENST00000446393.2;GENOMIC_POSITION=chr1:247775603;GENE_SYMBOL=OR9H1P
ENSG00000228336.2	5438	SNV-5438-T-C	T	C	.	.	TRANSCRIPT_ID=ENST00000446393.2;GENOMIC_POSITION=chr1:247775606;GENE_SYMBOL=OR9H1P
