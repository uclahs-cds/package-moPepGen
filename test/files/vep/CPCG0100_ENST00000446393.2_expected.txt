EVKQALR
EVKQALRR
IPTMQGC
IPTMQGCQK
IPTMQGRQK
IVSIFYTMVIPMLNPLIYSLR
IVSIFYTMVIPMLNPLIYSLRNK
LESLLHVR
LESLLHVRFPPNR
MLESLLHVR
MLESLLHVRFPPNR
MPESLLHVR
MPESLLHVRFPPNR
MRSTTFSVMSPHSCPSHVQIPL
MSPHSCPSHVQIPL
NKEVKQALR
PESLLHVR
PESLLHVRFPPNR
QALRRSMQK
RSMQKLSL
RSTTFSVMSPHSCPSHVQIPL
SMQKLSL
SPHSCPSHVQIPL
STTFSVMSPHSCPSHVQIPL
